Client/Frontend/Strings.swift

```diff
@@ -2056,6 +2056,36 @@ extension String {
         value: "Cannot Open Page",
         comment: "Title of the message shown when the user attempts to navigate to an invalid link.",
         lastUpdated: .unknown)
+    public static let CouldntDownloadWallpaperErrorTitle = MZLocalizedString(
+        "Wallpaper.Download.Error.Title.v106",
+        value: "Couldn’t Download Wallpaper",
+        comment: "The title of the error displayed to a user when they try change a wallpaper that failed downloading.",
```

Comment:
All these comment seem overly complicated?

```suggestion
        comment: "The title of the error displayed if download fails when changing a wallpaper.",
```

Similar for the next one.